zuul.d/project-templates.yaml

```diff
@@ -557,6 +557,7 @@
 - project-template:
     name: ansible-collections-community-vmware
     check:
+      queue: integrated
```

Comment:
we don't want this, we want to update https://github.com/ansible/ansible-zuul-jobs/blob/master/zuul.d/project-templates.yaml#L2 like you did here.

THis would group all jobs the same in check and gate in the integrated change queue.